Extension/src/LanguageServer/extension.ts

```diff
@@ -601,7 +496,7 @@ function applyUpdate(buildInfo: BuildInfo, updateChannel: string): Promise<void>
                 break;
             }
             try {
-                await installVsix(vsixPath, updateChannel);
+                await vscode.commands.executeCommand('workbench.extensions.installExtension', vscode.Uri.file(vsixPath));
```

Comment:
Do we need to update package.json to use this, or is it already available in the version of VS Code we're targeting?

Reply:
According to Sean (in the issue); They added it to 1.33.0.